### PR TITLE
T1542

### PR DESCRIPTION
**Pull request by humphd**

### CODE REVIEW

File: templates/supported/shared/plugins/speak/popcorn.speak.js [hunk 1]
```diff
@@ -82,10 +86,12 @@
         if( !options.pluginPath ) { options.pluginPath = "js/plugins/speak/"; }
 
         // SPEAK.JS by @kripken https://github.com/kripken/speak.js ---------------------------
-        try {
-          speakWorker = new Worker( options.pluginPath + 'speakWorker.js');
-        } catch(e) {
-          console.log('speak.js warning: no worker support');
+        if( !speakWorker ){
+          try {
+            speakWorker = new Worker( options.pluginPath + 'speakWorker.js');
```
Comment (secretrobotron) [inline templates/supported/shared/plugins/speak/popcorn.speak.js]: Just add some spacing before the last bracket here.

File: templates/supported/shared/plugins/speak/popcorn.speak.js [hunk 2]
```diff
@@ -82,10 +86,12 @@
...
+          } catch(e) {
+            console.log('speak.js warning: no worker support');
```
Comment (secretrobotron) [inline templates/supported/shared/plugins/speak/popcorn.speak.js]: And some spacing here.

File: templates/supported/shared/plugins/speak/popcorn.speak.js [hunk 3]
```diff
@@ -229,5 +235,6 @@
           options._target.removeChild( options.showTextEl );
         }
       }
-  });
-})( Popcorn );
+    };
+  }()))
```
Comment (secretrobotron) [inline templates/supported/shared/plugins/speak/popcorn.speak.js]: Missin' semicolon.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/69bc9e1e47f2c94/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @ScottDowne received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/6a6dd67c2170ff3/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/69bc9e1e47f2c94/output.txt

Total script time: 0.52 mins

Server is running on http://pr509.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/6a6dd67c2170ff3/output.txt

Total script time: 0.24 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/0a814a88f2bdf90/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/0a814a88f2bdf90/output.txt

Total script time: 0.21 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/b847dadbb53def4/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/b847dadbb53def4/output.txt

Total script time: 0.59 mins

Server is running on http://pr509.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/88ebded729f9bfb/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/88ebded729f9bfb/output.txt

Total script time: 0.55 mins

Server is running on http://pr509.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/4725122082ea8ff/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @ScottDowne received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/d605e68056582fd/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/4725122082ea8ff/output.txt

Total script time: 0.51 mins

Server is running on http://pr509.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/d605e68056582fd/output.txt

Total script time: 0.31 mins

**Pull request merged**